presto-docs/src/main/sphinx/release/release-0.243.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.243
+=============
+
+**Highlights**
+==============
+* Add function ``truncate(x, n)``.
+* Add aggregation function ``approxMostFrequent``.
+* Add session property ``check_access_control_on_utilized_columns_only``, which, when enabled, only performs access control checks on columns that would actually be required to produce the query output, ignoring columns that are referenced in the query, but are not required to compute the query results.
+* Add session property ``remote_functions_enabled`` to configure whether remote functions are allowed.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add function ``truncate(x, n)``.
+* Add aggregation function ``approxMostFrequent``.
+* Fix query failure when explaining ``Use`` statement.
+* Add session property ``check_access_control_on_utilized_columns_only``, which, when enabled, only performs access control checks on columns that would actually be required to produce the query output, ignoring columns that are referenced in the query, but are not required to compute the query results.
+* Add session property ``remote_functions_enabled`` to configure whether remote functions are allowed.
```

Comment:
what's a remote function?  Do we have documentation about this anywhere?

Reply:
Yes, lets link to an issue/pr/documentation that has more details on this.

Reply:
Looks like we dont want to expose this feature to users yet. Talked with @rongrong  and she suggested i remove this from the release note.